README.md

```diff
@@ -61,7 +65,7 @@ Contributions are very welcome! Check out the [Contribution Guidelines](/CONTRIB
 ## How is this Module versioned?
```

Comment:
You missed the link to the `Contribution Guidelines` right above this section.

Reply:
And `Gruntwork Philosophy` in the "Who maintains this module" section.

Reply:
Looks like there is no philosophy doc at all 😄